src/backup.rs

```diff
@@ -71,15 +71,15 @@ impl BackupEngine {
         }
     }
 
-    pub fn purge_old_backups(&mut self, num_backups_to_keep: usize) -> Result<(), Error> {
-        unsafe {
-            ffi_try!(ffi::rocksdb_backup_engine_purge_old_backups(
-                self.inner,
-                num_backups_to_keep as uint32_t
-            ));
-            Ok(())
-        }
-    }
+    //    pub fn purge_old_backups(&mut self, num_backups_to_keep: usize) -> Result<(), Error> {
+    //        unsafe {
```

Comment:
Is that still needed? I'm highly against leaving commented code without additional explanations.